person.rb

```diff
@@ -0,0 +1,31 @@
+# Represents the super class person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age = 18, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] The `age` parameter should be required, so it can not be optional. Only name and parent_permission are optional.

  ```
  def initialize(id, age, name = 'Unknown', parent_permission: true)
  ```

- [ ] **[OPTIONAL]** We can auto-generate the `id` rather than having to include it everytime in the list of arguments.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = Random.rand(1...1000)
  ```